technical/indicators/indicators.py

```diff
@@ -1260,3 +1260,28 @@ def tv_hma(dataframe: DataFrame, length: int = 9, field="close") -> DataFrame:
     dataframe.drop("h", inplace=True, axis=1)
 
     return dataframe
+
+
+def rma(dataframe: DataFrame, length: int = 15, field="close") -> DataFrame:
+    """
+    Source: Tradingview rma
+    Pinescript Author: Unknown
+
+    Args :
+        dataframe : Pandas Dataframe
+        length : RMA length
+        field : Field to use for the calculation
+
+    Returns :
+        dataframe : Pandas DataFrame with new column 'tv_rma'
+    """
+    alpha = 1.0 / length
+    series = dataframe[field]
+
+    for i in range(1, series.size):
+        series.iloc[i] = series.iloc[i] * alpha + (1 - alpha) * (
+            series.iloc[i - 1] if not isna(series.iloc[i - 1]) else 0
+        )
```

Comment:
uh - that'll be ... VERY slow in backtesting (it's a loop ...).

Are you sure there isn't a vectorized method to mirror this behaviour?

this will be ok for small timeranges - but when you have 10.000 candles in a backtest, it'll become very difficult to run.

At the very least (if it can't be vectorized) - please add a comment to the docstring noting that it'll be slow.